### ISSUE TITLE
update README for CGAL

### ISSUE THREAD

**Issue by steo85it** [sampotter/python-flux#54]
Just as a reminder, the [README]( https://github.com/sampotter/python-flux/blob/ff4c614d5f21bcf3e7c07cbe9ebf7f2b8b472181/README.md) needs to be updated to allow CGAL compile (and some parts clarified for embree, too).
(I can also do it, not assigning to anyone specific)

**Comment by steo85it** [comment]
plus, I would replace "==" by ">=" in requirements.txt, unless a specific (potentially old) version is really required (it's currently not the case).

**Comment by sampotter** [comment]
Yep please go ahead. If you put together a PR I'll be happy to take a look at it.
